src/design-tokens.yaml

```diff
@@ -134,6 +135,7 @@ background-color:
   danger: *color-red-30
   danger-dark: *color-red-60
   page-default: *color-white
+  light-blue: *color-blue-05
```

Comment:
Other colors here don't use the color name, just words like "light"/"dark". Should this perhaps be `info-extra-light` like the other two blue colors?

Reply:
Adam explicitly said it shouldn't be that, because it should not be associated with one of our statuses since it has nothing to do with them